### ISSUE TITLE
python-future changes link to outdated changelog

### ISSUE THREAD

**Issue by jkseppan** [dependabot/feedback#708]
A Dependabot PR to bump [python-future](https://github.com/PythonCharmers/python-future) to version 0.18.0 links to the project's changelog but shows no changes from it:

```
Bumps [future](https://github.com/PythonCharmers/python-future) from 0.17.1 to 0.18.0.
<details>
<summary>Changelog</summary>

*Sourced from [future's changelog](https://github.com/PythonCharmers/python-future/blob/master/docs/changelog.rst).*

</details>
<details>
<summary>Commits</summary>

- See full diff in [compare view](https://github.com/PythonCharmers/python-future/commits)
</details>
<br />
```

In fact, it looks like the project is not updating the changelog.rst file but the [whatsnew.rst](https://github.com/PythonCharmers/python-future/blob/master/docs/whatsnew.rst) file.

**Comment by feelepxyz** [comment]
@jkseppan thanks for reporting! Yeah looks like we don't support that file name yet, not sure I've seen that before.

@rebelagentm could be a good one to look at? We might just be able to add it to this list of files to check: https://github.com/dependabot/dependabot-core/blob/89c3ac4fcb39f089f982ba5bfe896fec3a961a7a/common/lib/dependabot/metadata_finders/base/changelog_finder.rb#L21

**Comment by rebelagentm** [comment]
@feelepxyz I'll take a look.

**Comment by rebelagentm** [comment]
@jkseppan This should be fixed now (v0.113.15).  Thanks again for reporting this!
